jujugui/static/gui/src/app/app.js

```diff
@@ -1907,14 +1912,18 @@ YUI.add('juju-gui', function(Y) {
           password: password
         });
       };
-      // Tell the environment to use the new socket URL when reconnecting.
-      this.env.set('socket_url', socketUrl);
+      if (reconnect) {
```

Comment:
Is there really any reason to not allow this to set the socket_url? Sure it'll be invalid but we aren't connecting so that's ok?